docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| hook               | Hook is the place within napari where we “promise” to call functions created by external developers and installed by the user.                                                                                                                                                                                                              |
+| hook specification | A function signature that defines the API that a plugin developer must adhere to when writing the function that we promise to call somewhere in the napari codebase.                                                                                                                                                                        |
+| keybinding         | The key or key combination that causes some functions to occur.                                                                                                                                                                                                                                                                             |
+| labels             | The labels layer is an image where pixel values are integers. Each unique integer represents a unique object, feature or classification.                                                                                                                                                                                                    |
+| layer              | Layers are the viewable objects that can be added to napari viewer. Currently napari supports the following layers: image, labels, points, shapes, surface, tracks, vectors.                                                                                                                                                                |
+| mousebinding       | The specific mouse click that causes some functions to occur.                                                                                                                                                                                                                                                                               |
+| plugin             | A napari plugin is a specialized Python package which extends napari's functionality through additional file i/o support, image analysis, or other utilities. Plugin developers specify this functionality through a combination of Python entry points and a specialized manifest file. Plugins for napari can be found at napari-hub.org. |
+| shapes             | The shapes layer allows you to display a list of NxD arrays, where each array corresponds to one shape, specified by N points in D coordinates. Currently supported shapes: rectangle, ellipses, polygons, paths, and lines. Can be used as a visualization tool to highlight ROI, or be used for downstream operations.                    |
```

Comment:
Need to link to the shapes page in the docs - this one is especially important because you need to know the specification for what format to describe your shape.

Also, I notice that there are several layer types missing from this glossary (image, vectors, tracks, points). For completeness, I think we should include them all.